### PR TITLE
Added the examples referenced by the 2.1 documentation set.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses the taipy-gui part of [taipy-doc/#379](https://github.com/Avaiga/taipy-doc/issues/379).

### CODE REVIEW

File: doc/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Taipy GUI Examples
+
+This directory contains an organized set of Python script files that can
```
Comment (FredLL-Avaiga) [inline doc/examples/README.md:3]: Should we mention the fact that these examples are used and linked fin the doc and add a back link?

File: doc/examples/charts/advanced-annotations.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# -----------------------------------------------------------------------------------------
+from taipy import Gui
+
+# Function to plot: x^3/3 - x
+def f(x):
+    return x*x*x/3-x
+
+
+# x values: [-2.2, ..., 2.2]
+x = [(x-10)/4.5 for x in range(0, 21)]
```
Comment (FredLL-Avaiga) [inline doc/examples/charts/advanced-annotations.py:24]: Too many x :-)

**Review by FredLL-Avaiga — APPROVED** [review]
Too good to be true

File: doc/examples/charts/advanced-annotations.py (hunk 2 above)
Comment (FabienLelaquais) [reply doc/examples/charts/advanced-annotations.py:24]: 😉

File: doc/examples/README.md (hunk 1 above)
Comment (FabienLelaquais) [reply doc/examples/README.md:3]: It is indicated that these are referenced from the doc.
However a back link is impossible due to the release branch name that changes...

File: doc/examples/README.md [hunk 3]
```diff
@@ -7,4 +7,4 @@ These script files are examples of how to build simple Taipy GUI applications,
 to demonstrate specific features of the taipy-gui package.
 
 These examples are organized by theme under this directory and are referenced
-from the Taipy documentation.
+from the [Taipy documentation](docs.taipy.io).
```
Comment (FredLL-Avaiga) [inline doc/examples/README.md]: Would not this be considered a relative URL ?
Does it need https:// ?

File: doc/examples/charts/bar-facing.py [hunk 4]
```diff
@@ -37,6 +38,7 @@
 data = {
   "Hobbies" : ["Archery", "Tennis", "Football", "Basket", "Volley", "Golf", "Video-Games", "Reading", "Singing", "Music"],
   "Female" : proportions_female,
+  # Negate these values so they appear to the left side
   "Male" : -proportions_male
```
Comment (FredLL-Avaiga) [inline doc/examples/charts/bar-facing.py:42]: I didn't know you could negate a list !

File: doc/examples/charts/bar-facing.py [hunk 5]
```diff
@@ -65,9 +67,8 @@
 
 page = """
 # Bar - Facing
-<|toggle|theme|>
 
-<|{data}|chart|type=bar|properties={properties}|width=60%|>
```
Comment (FredLL-Avaiga) [inline doc/examples/charts/bar-facing.py]: I approve the removal of width

File: doc/examples/charts/basics-multiple.py [hunk 6]
```diff
@@ -29,9 +29,8 @@
 
 page = """
 # Basics - Multiple traces
-<|toggle|theme|>
 
 <|{data}|chart|x=x|y[1]=y1|y[2]=y2|height=300px|>
```
Comment (FredLL-Avaiga) [inline doc/examples/charts/basics-multiple.py]: do you want to remove height as well ?

File: doc/examples/charts/basics-title.py [hunk 7]
```diff
@@ -24,9 +24,8 @@
 
 page = """
 # Basics - Title
-<|toggle|theme|>
 
 <|{[x*x for x in range(0, 11)]}|chart|title=Plotting x squared|layout={layout}|height=300px|>
```
Comment (FredLL-Avaiga) [inline doc/examples/charts/basics-title.py]: height

File: doc/examples/charts/radar-simple.py [hunk 8]
```diff
@@ -50,9 +50,8 @@
 
 md = """
 # Radar - Simple
-<|toggle|theme|>
 
 <|{data}|chart|type=scatterpolar|r=%|theta=Language|options={options}|layout={layout}|height=300%|>
```
Comment (FredLL-Avaiga) [inline doc/examples/charts/radar-simple.py]: 300% is a lot

**Review by FredLL-Avaiga — APPROVED** [review]
remove height ?

File: doc/examples/README.md (hunk 3 above)
Comment (FabienLelaquais) [reply doc/examples/README.md]: Of course it does!!!

**Comment by FabienLelaquais** [comment]
> remove height ?

Yeah I should.
Will do

Thanks

File: doc/examples/charts/radar-simple.py (hunk 8 above)
Comment (FabienLelaquais) [reply doc/examples/charts/radar-simple.py]: That will get dropped.

File: doc/examples/charts/basics-title.py (hunk 7 above)
Comment (FabienLelaquais) [reply doc/examples/charts/basics-title.py]: Will drop

File: doc/examples/charts/basics-multiple.py (hunk 6 above)
Comment (FabienLelaquais) [reply doc/examples/charts/basics-multiple.py]: Yes
Thx

File: doc/examples/charts/bar-facing.py (hunk 5 above)
Comment (FabienLelaquais) [reply doc/examples/charts/bar-facing.py]: 👍

File: doc/examples/charts/bar-facing.py (hunk 4 above)
Comment (FabienLelaquais) [reply doc/examples/charts/bar-facing.py:42]: That's not a list, but a NumPy array... 😉

**Pull request merged**